operatorcourier/cli.py

```diff
@@ -0,0 +1,63 @@
+from operatorcourier import courier_api
+import argparse
+import sys
+
+def main():
+    """Generate the CLI bits
+    """
+    _CliParser()
+
+class _CliParser():
+    """Class that generates the command line bits for the operator-courier cli tool
+    """
+
+    def __init__(self):
+        # Generate the command level parser
+        parser = argparse.ArgumentParser(
+            description='Build, verify and push operator bundles into external app registry',
+            usage='''operator-courier <command> [<args>]
+
+These are the commands you can use:
+    verify      Create a bundle and test it for correctness.
+    push        Create a bundle, test it, and push it to an app registry.
+''')
+        parser.add_argument('command', help='Subcommand to run')
+
+        args = parser.parse_args(sys.argv[1:2])
+        if not hasattr(self, args.command):
+            print('Unrecognized command')
+            parser.print_help()
+            exit(1)
```

Comment:
Please do `parser.error('error text')`

Reply:
Addressed in latest commit.

Reply:
with parser.error you don't need exit(1) or print_help, it will exit with SystemExit=2

Reply:
usage text will be printed automatically with parser.error